src/components/Shared/Slug.tsx

```diff
@@ -16,7 +16,7 @@ const Slug: FC<Props> = ({ slug, prefix, className = '' }) => {
       )}
     >
       {prefix}
-      {slug}
+      {slug.replace('.lens', '')}
```

Comment:
But why? 🤦🏻

Reply:
Technically??

If it's a product-related question, yes, it's a request from the Lens product team 🙇🏼